{
  "title": "CVE-2022-23097",
  "cve": "CVE-2022-23097",
  "notes": {
    "description": "An issue was discovered in the DNS proxy in Connman through 1.40. forward_dns_reply mishandles a strnlen call, leading to an out-of-bounds read.",
    "published": "2022-01-28",
    "modified": "2023-10-31"
  },
  "references": [
    {
      "url": "https://www.debian.org/security/2022/dsa-5231",
      "description": "DEBIAN:DSA-5231"
    },
    {
      "url": "https://security.gentoo.org/glsa/202310-21",
      "description": "GENTOO:GLSA-202310-21"
    },
    {
      "url": "https://git.kernel.org/pub/scm/network/connman/connman.git/log/",
      "description": "MISC:https://git.kernel.org/pub/scm/network/connman/connman.git/log/"
    },
    {
      "url": "https://www.openwall.com/lists/oss-security/2022/01/25/1",
      "description": "MISC:https://www.openwall.com/lists/oss-security/2022/01/25/1"
    },
    {
      "url": "https://lists.debian.org/debian-lts-announce/2022/02/msg00009.html",
      "description": "MLIST:[debian-lts-announce] 20220209 [SECURITY] [DLA 2915-1] connman security update"
    }
  ]
}
